.github/workflows/cdk8s-release.yaml

```diff
@@ -27,9 +27,10 @@ jobs:
       - name: Test
         run: |
           #!/bin/sh
-          cd examples
-          wing compile -t ../lib/index.js nodejs.main.w
-          wing compile -t ../lib/index.js ubuntu.main.w
+
+          wing compile -t ./lib/index.js examples/nodejs.main.w
```

Comment:
We could run the test.sh here instead, to avoid out-of-sync testing scenarios

Reply:
Sure, let me try that

Reply:
Done